src/ray/common/id.cc

```diff
@@ -139,6 +139,21 @@ const JobID ComputeJobId(const WorkerID &driver_id) {
   return JobID(driver_id);
 }
 
+const ActorHandleID ComputeNextActorHandleId(const ActorHandleID &actor_handle_id,
+                                             int64_t num_forks) {
+  // Compute hashes.
+  SHA256_CTX ctx;
+  sha256_init(&ctx);
+  sha256_update(&ctx, reinterpret_cast<const BYTE *>(actor_handle_id.Data()),
+                actor_handle_id.Size());
+  sha256_update(&ctx, reinterpret_cast<const BYTE *>(&num_forks), sizeof(num_forks));
+
+  // Compute the final task ID from the hash.
+  BYTE buff[DIGEST_SIZE];
+  sha256_final(&ctx, buff);
+  return ActorHandleID::FromBinary(std::string(buff, buff + ActorHandleID::Size()));
```

Comment:
Maybe add a check that `DIGEST_SIZE >= ActorHandleID::Size()`?

Reply:
Resolved in #5034 and merged to this PR.